spec/controllers/concerns/hyrax/works_controller_behavior_spec.rb

```diff
@@ -19,7 +19,8 @@
     include Hyrax::WorksControllerBehavior
 
     self.curation_concern_type = Hyrax::Test::SimpleWork
-    self.search_builder_class  = Hyrax::Test::SimpleWorkSearchBuilder
+    self.search_builder_class  = Wings::WorkSearchBuilder(Hyrax::Test::SimpleWork)
+    self.work_form_service     = Hyrax::FormFactory.new
```

Comment:
Would we need an instance for this? It appears that you could have a module_function on `Hyrax::FormFactory.build`

Reply:
Per @no-reply discussion, there is no "strong" need for an instance method. It does allow for greater extensibility.